clang-tools-extra/clang-tidy/daedalean/DaedaleanTidyModule.cpp

```diff
@@ -37,22 +38,19 @@ class DaedaleanModule : public ClangTidyModule {
   void addCheckFactories(ClangTidyCheckFactories &CheckFactories) override {
     CheckFactories.registerCheck<AssignmentOperatorsCheck>(
         "daedalean-assignment-operators");
-    CheckFactories.registerCheck<ClassMethodsCheck>(
-        "daedalean-class-methods");
+    CheckFactories.registerCheck<ClassMethodsCheck>("daedalean-class-methods");
```

Comment:
Why this file reformatted? It is managed by scripts to add new checks

Reply:
I run clang-format on save by default. I will leave it as it was